docs/install.md

```diff
@@ -0,0 +1,150 @@
+for both methods of installation.
+
+PDO requires installation of the SDK for Intel Software Guard Extensions
+(SGX). SGX can be used either in simulation mode or hardware
+mode. Simulation mode will work even when there is no processor support
+for SGX. However, simulation mode is not secure and is suitable soley
+for development and demonstration. Hardware mode, which uses trusted
+execution capabilities in the processor to protect confidentiality and
+integrity of computation, requires appropriate processor
+support. Information about supported processors is provided below.
+
+## Recommended Host System
+
+The recommended host-system configuration for Private Data Objects is to
+separate the Private Data Objects components from the Sawtooth components. This
+means (at least) two different physical systems if using SGX-enabled hardware.
+If running in SGX simulation mode, this could be two virtual machines or
+containers.
+
+Sawtooth (and the PDO transaction processors for Sawtooth) should be run on
+Ubuntu 16.04.
+
+Private Data Objects services (specifically the enclave service, provisioning
+service, and the client) should be run on Ubuntu 18.04. PDO has been tested on
+Ubuntu 16.04, 17.10, and 18.04.
+
+Sawtooth and PDO may run on other Linux distributions, but the installation
+process is likely to be more complicated, and the use of other distributions is
+not supported by their respective communities at this time.
+
+## <a name="SGX">Intel Software Guard Extensions (SGX)</a>
+
+Private Data Objects uses the trusted execution capabilities provided by
+Intel SGX to protect integrity and confidentiality. More information
+about SGX can be found on the
+[Intel SGX website](https://software.intel.com/en-us/sgx) including
+[detailed installation instructions](https://download.01.org/intel-sgx/linux-2.4/docs/Intel_SGX_Installation_Guide_Linux_2.4_Open_Source.pdf).
+
+SGX can operate in either simulation mode or hardware mode. Simulation
+mode does not require any processor support for SGX and can be useful
+for development and testing. However, simulation mode does not provide
+any protection for confidential data and does not guarantee integrity of
+execution. To use SGX in simulation mode set the `SGX_MODE` environment
+variable to `SIM`:
+
+```bash
+export SGX_MODE=SIM
+```
+
+SGX hardware mode uses capabilities for trusted execution in the
+processor to protect confidentiality and integrity of computation. SGX
+hardware mode requires processor support for SGX
+([commonly available on recent Intel processors](https://ark.intel.com/content/www/us/en/ark/search/featurefilter.html)).
+To use SGX in hardware mode set the `SGX_MODE` environment variable to
+`HW`:
+
+```bash
+export SGX_MODE=HW
+```
+
+The remainder of this section provides information about preparing to
+run Private Data Objects using SGX in hardware mode. Specifically, there
+are steps that must be taken to enable attestation of the hardware
```

Comment:
Would it be possible to add section numbers? Not so easy to figure out where the section ends, and where the next begins. 

(as of now, there is a very faint line below new sections)

Reply:
i would prefer not. this is pretty standard markdown.